### PR TITLE
Fix CSS for generic font families

### PR DESCRIPTION
**Pull request by d07RiV**
Issue #2701

### CODE REVIEW

File: src/modules/chat_font_settings/index.js [hunk 1]
```diff
@@ -12,9 +12,17 @@ Leave the field blank to use the default.`;
 
 const STYLE_ID = 'bttv-font-size';
 
+const encodeFontFamily(fontFamily) => {
```
Comment (night) [inline src/modules/chat_font_settings/index.js]: this is invalid JavaScript

File: src/modules/chat_font_settings/index.js [hunk 2]
```diff
@@ -12,9 +12,17 @@ Leave the field blank to use the default.`;
...
+    if (['serif', 'sans-serif', 'monospace', 'cursive', 'fantasy', 'system-ui'].indexOf(fontFamily)) {
+        return html.escape(fontFamily);
+    } else {
+        return '"' + html.escape(fontFamily) + '"';
```
Comment (night) [inline src/modules/chat_font_settings/index.js]: you should use template literals

File: src/modules/chat_font_settings/index.js [hunk 3]
```diff
@@ -12,9 +12,17 @@ Leave the field blank to use the default.`;
 
 const STYLE_ID = 'bttv-font-size';
 
+const encodeFontFamily(fontFamily) => {
+    if (['serif', 'sans-serif', 'monospace', 'cursive', 'fantasy', 'system-ui'].indexOf(fontFamily)) {
```
Comment (night) [inline src/modules/chat_font_settings/index.js]: indexOf returns `-1` when it doesn't find anything .. this evaluates to true when it doesn't find anything

Comment (night) [reply src/modules/chat_font_settings/index.js]: there should also be a constant made of all generic names

File: src/modules/chat_font_settings/index.js (hunk 2 above)
Comment (night) [reply src/modules/chat_font_settings/index.js]: the logic can also be improved by always escaping the passed font family and then choosing to wrap it in quotes if it is not a generic name.

**Comment by d07RiV** [comment]
Whoops that's embarassing, fixed those. Are those compiled into ES5 btw?

**Pull request merged**